libc-test/semver/unix.txt

```diff
@@ -761,6 +761,8 @@ setpgid
 setsid
 setsockopt
 setuid
+setreuid
```

Comment:
nit: ideally in alphabetical order

Reply:
Good catch! I'm going to fix once this is merged.